pkg/controller/controller.go

```diff
@@ -320,6 +321,16 @@ func (ctrl *resizeController) syncPVC(key string) error {
 		return fmt.Errorf("expected volume but got %+v", volumeObj)
 	}
 
+	if ctrl.isFsResizeComplete(pvc, pv) && metav1.HasAnnotation(pv.ObjectMeta, util.AnnPreResizeCapacity) {
```

Comment:
I have renamed the function. I noticed that there are several places in the code that uses fsResize word. Will there be future issues to address this refactoring?

Reply:
There is still conditions on the PVC which use fsResize wording, we can't rename that because that has become part of API now. But other than that we could and should rename everything else.